src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -2,14 +2,16 @@
 
 import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.logfields.LogFieldsTable;
 import frc.robot.subsystems.elevator.io.ElevatorIO;
 import frc.robot.subsystems.elevator.io.ElevatorIOSparkMax;
 
 public class Elevator extends SubsystemBase{
-    private final ElevatorIO io = new ElevatorIOSparkMax(null);
+    private final LogFieldsTable fieldsTable = new LogFieldsTable ("Elevator");
+    private final ElevatorIO io = new ElevatorIOSparkMax(fieldsTable);
 
     public void setSpeed(double speed){
-        speed = MathUtil.clamp(speed, 0, 1);
+        speed = MathUtil.clamp(speed, 0, ElevatorConstants.SPEED_LIMIT);
```

Comment:
You also need to set the minimum limit. currently it's 0, meaning you can not move backward. you should change it to `-ElevatorConstants.SPEED_LIMIT`